docs/book/src/migration/v1vsv2.md

```diff
@@ -0,0 +1,61 @@
+# Kubebuilder v1 vs v2
+
+There are a number of changes in
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime),
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools)
+and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder).
+
+## Common changes
+
+- They have migrated from `dep` to `go modules` for managing dependencies.
+
+## Controller-runtime
+
+- Automatic certificate generation for webhooks has been removed, and webhooks
+will no longer self-register. Use controller-tools to generate a webhook
+configuration, and use something like cert-manager to generate your certificates
+(if you don't already have a solution). Kubebuilder supports them out-of-the-box.
+
+- If your API implements
```

Comment:
We should probably only list changes that affect existing users.  Maybe add a note about that at the top.

Reply:
Or maybe put new features elsewhere?  I don't want to have to update this doc every time something new comes out though.